.travis.yml

```diff
@@ -32,9 +32,13 @@ install:
     - export CEI_CMAKE="/home/travis/.local/bin/cmake"
     - python -m pip install --upgrade git+https://github.com/yt-project/yt.git
     - sudo curl -L -o /usr/local/bin/cmake-easyinstall https://git.io/JvLxY && sudo chmod a+x /usr/local/bin/cmake-easyinstall
+    - sudo cmake-easyinstall git-https://github.com/openPMD/openPMD-api.git
+        -DopenPMD_USE_PYTHON=OFF -DBUILD_TESTING=OFF -DBUILD_EXAMPLES=OFF
+        -DCMAKE_INSTALL_PREFIX=/usr/local
```

Comment:
Could we make this conditional? It is not required for `WARPX_CI_REGULAR_CARTESIAN`, so if we can spare a few minutes there that's always good.

Reply:
sure, this will save 2min.

Reply:
Indeed, more that 2 minutes on a job that takes >40 min, where the limit is 50 min.

Reply:
fully agree, no irony implied :)

Reply:
Oops, sorry :disappointed:.
Thanks!

Reply:
I should not have written "sure", sorry for that :)